bitbot/8ball_test.go

```diff
@@ -0,0 +1,73 @@
+package bitbot
```

Comment:
This is a false positive, because then we don't get to access bitbot's internal functions, the ones we are testing.
I don't understand if this means I should be writing these tests another way, maybe by exporting all functions we test ?
While this means that we only care about the API, and not the internal implementation, this also makes testing triggers much harder.

Reply:
After discussing the matter, it has been decided that a testing package wasn't needed.